src/routes/about/+page.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+    import Contact from '$lib/components/Contact.svelte';
+    import MissionVision from './MissionVision.svelte';
+    import SectionHeaderTextContainer from './SectionHeaderTextContainer.svelte';
+</script>
+
+<section>
+    <SectionHeaderTextContainer />
+</section>
+<section>
+    <MissionVision />
+</section>
+<section>
+    <Contact />
+</section>
```

Comment:
I like the use of `section` here, but I think they belong in the component level, not in the page level. Let's put the `section` as the root element (of the respective components) rather than the `div`.